### DISCUSSION TITLE
Flatpak crashing

### DISCUSSION THREAD

**Discussion by SillySwitch** [flyinghead/flycast#1099] [General]
My friend and I are unable to run Flycast at this time 
No matter what game we load it crashes Flycast immediately. 
My friend is using Manjaro, and I am using Pop_OS so this is not specific to a certain distro 

I have tried uninstalling and reinstalling a fresh Flycast from the Flathub, and this issue is still there. 
We have tried local games and games on secondary, and it provides the same issue, it will also not load a bios when tested our bios.

My friend was able to use it in the app image version 

This is the error message we get in the terminal when trying to run 

"00:00:436 sdl/sdl.cpp:599 N[RENDERER]: Monitor refresh rate: 144 Hz (1920 x 1080)
00:00:438 rend/gui.cpp:294 N[RENDERER]: Screen DPI is 142, size 640 x 480. Scaling by 1.06
00:00:509 rend/gles/gles.cpp:562 N[RENDERER]: OpenGL version 4.6
00:02:851 hw/mem/addrspace.cpp:431 N[VMEM]: Info: nvmem is enabled
00:02:860 hw/mem/addrspace.cpp:462 N[VMEM]: BASE 0x7f31f5b80000 RAM(16 MB) 0x7f3201b80000 VRAM64(8 MB) 0x7f31f9b80000 ARAM(2 MB) 0x7f3215b80000
00:02:867 hw/mem/addrspace.cpp:431 N[VMEM]: Info: nvmem is enabled
00:02:874 hw/mem/addrspace.cpp:462 N[VMEM]: BASE 0x7f31f5b80000 RAM(16 MB) 0x7f3201b80000 VRAM64(8 MB) 0x7f31f9b80000 ARAM(2 MB) 0x7f3215b80000
00:02:881 emulator.cpp:54 N[BOOT]: Game ID is []
00:02:882 emulator.cpp:194 W[BOOT]: No region specified in IP.BIN
00:02:882 sdl/sdl_gamepad.h:373 N[INPUT]: Resetting SDL gamepad to default: 0 1
/usr/include/c++/12.2.0/bits/stl_vector.h:1123: std::vector<_Tp, _Alloc>::reference std::vector<_Tp, _Alloc>::operator[](size_type) [with _Tp = PolyParam; _Alloc = std::allocator<PolyParam>; reference = PolyParam&; size_type = long unsigned int]: Assertion '__n < this->size()' failed."

![Screenshot from 2023-06-03 22-25-19](https://github.com/flyinghead/flycast/assets/134230920/306fdaee-d588-46a1-b428-d8587465606e)

**Comment by SillySwitch** [comment]
My friend also shared a screen shot of their error on Manjaro, we seem to both be getting the same error as well 

![image](https://github.com/flyinghead/flycast/assets/134230920/1793b525-8bee-4fa6-b78e-324cda1475d9)

**Comment by flyinghead** [comment]
See #1100
